config/routes.rb

```diff
@@ -84,6 +84,8 @@
       post "/publish", on: :member, to: "providers#publish"
       get "/training-providers", on: :member, to: "providers#training_providers"
       get "/training-providers-courses", on: :member, to: "training_providers_courses#index", as: "download_training_providers_courses"
+      get "/visas/edit", to: "providers/visas#edit"
```

Comment:
Could we just use `/visas` as the path, as I can't see us ever needing a separate page for a `show` action just for visas?

```suggestion
      get "/visas", to: "providers/visas#edit"
```